porcelain/wallet_test.go

```diff
@@ -0,0 +1,42 @@
+package porcelain_test
+
+import (
+	"context"
+	"testing"
+
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmRXf2uUSdGSunRJsM9wXSUNVwLUGCY3So5fAs7h2CBJVf/go-hamt-ipld"
+
+	"github.com/filecoin-project/go-filecoin/actor"
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/porcelain"
+	"github.com/filecoin-project/go-filecoin/state"
+	"github.com/filecoin-project/go-filecoin/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type walletTestPlumbing struct{}
+
+func (wtp *walletTestPlumbing) ChainLatestState(ctx context.Context) (state.Tree, error) {
+	store := hamt.NewCborStore()
+	tree := state.NewEmptyStateTree(store)
+
+	testActor := actor.NewActor(cid.Undef, types.NewAttoFILFromFIL(20))
```

Comment:
This would be a bit more flexible for future tests with the balance as a field on the struct, and the constant `20` mentioned only in the test method.